modules/signals/src/index.ts

```diff
@@ -10,3 +10,4 @@ export { withComputed } from './with-computed';
 export { withHooks } from './with-hooks';
 export { withMethods } from './with-methods';
 export { withState } from './with-state';
+export { DeepSignal } from './deep-signal';
```

Comment:
we could move this export to the top of the file to keep the alphabetical order

Reply:
👍 sounds good, thanks.